src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```
@@ -100,6 +104,9 @@ else()
     find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.6/include")
     find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.5/include")
     #FreeBSD
+    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm70/include")
+    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm60/include")
+    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm50/include")
```

In FreeBSD 12, ports had llvm4 package as well, could that be specified as well:

```suggestion
    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm50/include")
    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm40/include")
```

---

In FreeBSD 12, ports had llvm4 package too, could that be specified as well:

```suggestion
    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm50/include")
    find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm40/include")
```